### PR TITLE
fix(codemods): Codemods create extra parentheses 

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

Sometimges codemods are adding extra parentheses around component and it should not.

There is a bug in `recast` library used by `jscodeshift`(codemod library) casing this behaviour.

### Additional context

It will happen when there is `return` statement in the component.

More info can be found here:
https://github.com/facebook/jscodeshift/issues/534
https://github.com/benjamn/recast/pull/1406

### Issue reference

[Codemods are adding extra parentheses](https://jira.almacareer.tech/browse/DS-1529)

### CODE REVIEW

File: packages/codemods/src/helpers/index.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { errorMessage, infoMessage, logMessage } from './message';
 import { _dirname } from './path';
+import { removeCircleBrackets } from './removeCircleBrackets';
 
-export { _dirname, errorMessage, infoMessage, logMessage };
+export { _dirname, errorMessage, infoMessage, logMessage, removeCircleBrackets };
```
Comment (literat) [inline packages/codemods/src/helpers/index.ts:5]: **suggestion (blocking):** Please use correct naming.

Types of Brackets:
- Parentheses ( )
- Square brackets [ ]
- Curly brackets { }
- Angle brackets ⟨ ⟩

My suggestion is to use `removeParentheses`.

File: packages/codemods/src/transforms/v3/web-react/__testfixtures__/button-isSquare-prop-name.input.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```
Comment (literat) [inline packages/codemods/src/transforms/v3/web-react/__testfixtures__/button-isSquare-prop-name.input.tsx:1]: **question (if-minor):** Why is this added?

File: packages/codemods/tsconfig.json [hunk 3]
```diff
@@ -25,8 +25,9 @@
     "skipLibCheck": true,
     "typeRoots": ["../../node_modules/@types"],
     "types": ["node", "jest"],
-    "module": "es2020"
+    "module": "es2020",
+    "target": "es2020"
   },
-  "include": ["./", "./.eslintrc.js"],
+  "include": ["./src/**/*"],
```
Comment (literat) [inline packages/codemods/tsconfig.json:31]: **note (if-minor):** I think we want to include all files, don't we?

File: packages/codemods/src/transforms/v3/web-react/__testfixtures__/button-isSquare-prop-name.input.tsx (hunk 2 above)
Comment (pavelklibani) [reply packages/codemods/src/transforms/v3/web-react/__testfixtures__/button-isSquare-prop-name.input.tsx:1]: i am getting errors everywhere: `'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.`

Comment (literat) [reply packages/codemods/src/transforms/v3/web-react/__testfixtures__/button-isSquare-prop-name.input.tsx:1]: ESLint problem? I think we want to ignore the fixtures or have agreed to do so, haven't we? If so, I can solve this separately.

**Review by literat — APPROVED** [review]

**Review by curdaj — APPROVED** [review]
